### PR TITLE
Fix for plugins' packages file not updating packages

### PR DESCRIPTION
**Pull request by IAmSilK**

### CODE REVIEW

File: framework/OpenMod.Core/Plugins/PluginAssemblyStore.cs [hunk 1]
```diff
@@ -149,6 +149,11 @@ public async Task<ICollection<Assembly>> LoadPluginAssembliesAsync(IPluginAssemb
                     //Remove Assembly from loading because required dependencies are missing.
                     providerAssemblies.Remove(providerAssembly);
 
+                    foreach (var loaderException in ex.LoaderExceptions)
+                    {
+                        m_Logger.LogDebug(loaderException, "Exception occurred when getting types in plugin assembly");
```
Comment (Trojaner) [inline framework/OpenMod.Core/Plugins/PluginAssemblyStore.cs:154]: Please log assembly name as well

**Review by Trojaner — CHANGES_REQUESTED** [review]
Must support `latest` as version in packages.yaml

File: framework/OpenMod.Core/Plugins/PluginAssemblyStore.cs (hunk 1 above)
Comment (IAmSilK) [reply framework/OpenMod.Core/Plugins/PluginAssemblyStore.cs]: Resolved by eee27bf

**Comment by rube200** [comment]
I tested with standalone and it seems to work fine.
packages.yaml
```
packages:
- id: OpenMod.Economy
  version: latest
```

**Review by rube200 — COMMENTED** [review]
Should be fixed now

**Review by Trojaner — APPROVED** [review]

**Pull request merged**